starters/express-apollo-prisma/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "endOfLine": "lf",
```

Comment:
Can go in the `.editorconfig`, prettier reads the that config as well and sets its own configuration to match.